openpos-client-libs/projects/openpos-client-core-lib/src/lib/screens-with-parts/options/options-screen-dialog.component.html

```diff
@@ -1,8 +1,8 @@
 <div class="options-outer" responsive-class >
     <app-dialog-header class="options-header"></app-dialog-header>
     <div class="options-body">
-        <img *ngIf="screen.imageUrl" [src]="screen.imageUrl | imageUrl" class="options-image">
-        <app-instructions *ngIf="screen.prompt" [instructions]="screen.prompt" [instructionsSize]="'text-sm'"></app-instructions>
+        <img *ngIf="screen && screen.imageUrl" [src]="screen.imageUrl | imageUrl" class="options-image">
```

Comment:
This does the same thing but shorter
```suggestion
        <img *ngIf="screen?.imageUrl" [src]="screen.imageUrl | imageUrl" class="options-image">
```